assets/src/edit-story/components/canvas/carousel/index.js

```diff
@@ -187,6 +192,10 @@ const GridViewContainer = styled.div`
   pointer-events: all;
 `;
 
+const SafeModeButton = styled(SafeMode)`
```

Comment:
Now this is just a styled SVG. For accessibility reasons, this needs to be a proper button.

```suggestion
const SafeModeButton = styled.button`
```

And then further down you can do:

```js
<SafeModeButton>
<SafeMode />
</SafeModeButton>
```